modules/external-api/src/main/java/org/opencastproject/external/util/statistics/DataResolutionUtils.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package org.opencastproject.external.util.statistics;
+
+import org.opencastproject.statistics.api.DataResolution;
+
+import org.json.simple.JSONArray;
+
+import java.util.Optional;
+import java.util.Set;
+
+public final class DataResolutionUtils {
+
+  private DataResolutionUtils() {
+  }
+
+  public static JSONArray toJson(Set<DataResolution> dataResolutions) {
+    JSONArray result = new JSONArray();
+    for (DataResolution resolution : dataResolutions) {
+      result.add(toString(resolution));
+    }
+    return result;
+  }
+
+  public static String toString(DataResolution dataResolution) {
+    return dataResolution.toString().toLowerCase();
```

Comment:
Instead of creating a lot of utility classes, why can't this kind of functionality be part of the classes, here DataResolution, themselves? They are a lot harder to miss that way, so chances are slimmer that people create duplicate utility functionality, as often seen in opencast...

Reply:
... because changing the classes in the statistics service would imply changing the External API which makes it hard to detect unintentional changes to the External API.
At least for the representation of data, the External API should really use its own code to create representations as not having dependencies to shared code seems more important than to avoid code duplication in the case of the External API. 

In this particular example, though, the External API is not actually changed when messing around with DataResolution as the exact values of data resolutions are not part of the External API specification.